libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid.service.ts

```diff
@@ -292,6 +295,19 @@ export class SkyAgGridService implements OnDestroy {
       [SkyCellClass.Invalid]: getValidatorFn(),
     };
 
+    function getHeaderClass(
+      ...classNames: string[]
+    ): (params: HeaderClassParams) => string | string[] | undefined {
+      return (params: HeaderClassParams): string | string[] | undefined => {
+        const minWidth = Number(params.column?.getMinWidth());
+        const maxWidth = Number(params.column?.getMaxWidth());
+        if (params.column?.isResizable() && minWidth < maxWidth) {
```

Comment:
It looks like further down we define `minWidth` and `maxWidth` as undefined by default. `undefined < undefined` is false. Does this mean that in order to get the resizing styling consumers must set min and max width?